modules/swagger-parser/src/main/java/io/swagger/parser/processors/ExternalRefProcessor.java

```diff
@@ -33,6 +33,12 @@ public ExternalRefProcessor(ResolverCache cache, Swagger swagger) {
     public String processRefToExternalDefinition(String $ref, RefFormat refFormat) {
         final Model model = cache.loadRef($ref, refFormat, Model.class);
 
+        if(model == null) {
+            // stop!  There's a problem
+            LOGGER.warn("unable to load model reference from `" + $ref + "`.  It may not be available " +
+                    "or the reference isn't a valid model schema");
+            return $ref;
```

Comment:
This is halfway to the goal, it doesn't die, but it's not nice in that there's no message sent back to the end user